std_execution.bs

```diff
@@ -5161,197 +5161,86 @@ are all well-formed.
-
-#### `execution::let_stopped` <b>[exec.let_stopped]</b> #### {#spec-execution.senders.adapt.let_stopped}
-
-1. `execution::let_stopped` is used to insert continuations creating more work dependent on the error its input senders into a sender chain.
-
-2. The name `execution::let_stopped` denotes a customization point object. For some subexpressions `s` and `f`, let `S` be `decltype((s))`, let `F` be the decayed type of `f`, and let `f'` be an xvalue that refers to an object decay-copied from `f`. If `S` does not satisfy `execution::sender` or if `F` does not satisfy `invocable`, `execution::let_stopped` is ill-formed. Otherwise, the expression
-    `execution::let_stopped(s, f)` is expression-equivalent to:
-
-    1. `tag_invoke(execution::let_stopped, get_completion_scheduler<set_value_t>(s), s, f)`, if that expression is valid.
-
-        * <i>Mandates:</i> The type of the `tag_invoke` expression above satisfies `execution::sender`.
-
-    2. Otherwise, `tag_invoke(execution::let_stopped, s, f)`, if that expression is valid.
-
-        * <i>Mandates:</i> The type of the `tag_invoke` expression above satisfies `execution::sender`.
-
-    3. Otherwise, given a receiver `out_r` and an lvalue `out_r'` refering to an object decay-copied from `out_r`.
-    
-        1. Let `r` be an rvalue of a receiver type `R` such that:
-    
-            1. `execution::set_value(r, args...)` is expression-equivalent to `execution::set_value(std::move(out_r'), args...)`.
-
-            2. `execution::set_error(r, e)` is expression-equivalent to `execution::set_value(std::move(out_r'), e)`.
-
-            3. When `execution::set_stopped(r)` is called, calls `invoke(f')`, resulting in a sender `s3`. It then calls `execution::connect(s3, std::move(out_r'))`, resulting in an operation state
-                `op_state3`. `op_state3` is saved as a part of `op_state2`. It then calls `execution::start(op_state3)`. If any of these throws an exception, it catches it and calls `execution::set_error(std::move(out_r'), current_exception())`. If any of these expressions would be ill-formed, `execution::set_stopped(r)` is ill-formed.
-
-        2. `execution::let_stopped(s, f)` returns a sender `s2` such that:
-        
-            1. If the expression `execution::connect(s, r)` is ill-formed, `execution::connect(s2, out_r)` is ill-formed.
-            
-            2. Otherwise, let `op_state2` be the result of `execution::connect(s, r)`. `execution::connect(s2, out_r)` returns an operation state `op_state` that stores `op_state2`. `execution::start(op_state)` is expression-equivalent to `execution::start(op_state2)`.
+                1. We denote by the expression <code><i>completion-sigs</i>&lt;Snd, Env></code> the set set of signature types in the <code><i>type-list</i></code> equivalent to the template arguments of the `completion_signatures` class returned by `execution::get_completion_signatures(declval<Snd>(), declval<Env>)`.
```

Comment:
I'm not sure if this is the most accurate way to describe the operation of getting the associated signatures list for a sender (and it's env). But, I think it captures the intent.

Reply:
I think this is insufficient, but it probably doesn't matter since the fix will come in #410. For now, can you just fix this:

```suggestion
                1. We denote by the expression <code><i>completion-sigs</i>&lt;Snd, Env></code> the set of signature types in the <code><i>type-list</i></code> equivalent to the template arguments of the `completion_signatures` class returned by `execution::get_completion_signatures(declval<Snd>(), declval<Env>)`.
```